docs/src/main/tut/plots.md

```diff
@@ -73,79 +73,101 @@ EvilPlot takes an unusual approach to overlaying, in the spirit of building more
 ones. When you want to make a multilayered plot, just make a plot for each layer. We then give you a combinator,
 `Overlay`, to compose all the plots into a meaningful composite.
 
-Let's look at two equally meaningful plots. The first is a
-[Ramachandran plot](https://en.wikipedia.org/wiki/Ramachandran_plot), which is a plot of the phi and psi dihredral
-angles in a peptide bond. This data came from a molecular dynamics simulation of alanine dipeptide, so we have a
-computed "phi" and "psi" from the simulation for each iteration. We want to look at how these two angles were
-distributed, so let's make a contour plot out of the density.
+Let's take a look at some real world data. In its natural state, real world data can be hard to interpret and even harder
+to put to use. Take a look at the public data from https://data.boston.gov/dataset/wicked-free-wi-fi-locations
+containing the Latitude(y) and Longitude(x) coordinates of several free wifi hotspots in Boston. To start, lets parse this
+file into a `Seq[Points]`:
+
+```scala
+import com.cibo.evilplot.numeric.Point
+import scala.collection.mutable.ArrayBuffer
+import scala.io.Source
+import scala.util.Try
+
+val path: String = //Path to saved CSV file
+val data: Seq[Point] = {
+    val points: ArrayBuffer[Point] = ArrayBuffer[Point]()
+    val bufferedSource = Source.fromFile(path + "wifi.csv")
+    //Drop the header row, we just need the points
+    for (line <- bufferedSource.getLines.drop(1)) {
+      val cols: Array[String] = line.split(",").map(_.trim)
+      //Columns 0 and 1 contain the x/y coordinates
+      points += Point(cols(0).toDouble, cols(1).toDouble)
+    }
+    bufferedSource.close()
+    points
+}
+```
+
+Now that we have our data in a usable form, lets make a Contour plot to show the general density of free wifi locations
```

Comment:
"lets" -> "let's"
"Contour" -> "contour"
"wifi" -> "WiFi" or "Wi-Fi"

(I think there are a couple other instances of "wifi")